comp/core/gui/guiimpl/gui.go

```diff
@@ -106,13 +106,15 @@ func newGui(deps dependencies) (optional.Option[guicomp.Component], error) {
 	// Create a CSRF token (unique to each session)
 	e := g.createCSRFToken()
 	if e != nil {
-		return optional.NewNoneOption[guicomp.Component](), e
+		g.logger.Errorf("GUI server initialization failed (unable to create CSRF token): ", e)
+		return optional.NewNoneOption[guicomp.Component](), nil
```

Comment:
As this function never returns an error, you can update the signature to `newGui(deps dependencies) (optional.Option[guicomp.Component]`